.babelrc

```diff
@@ -0,0 +1,14 @@
+{
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react"
+  ],
+  "plugins": [
+    "@babel/plugin-transform-react-jsx",
+    ["transform-jsx-classnames",
+      {
+        "dedupe": true
+      }
+    ]
```

Comment:
Sounds like we don't need this anymore. Yay!

Reply:
Yeah, can probably pull that out now